### ISSUE TITLE
Einlagern/Sammeln lässt sich nicht sauber stoppen

### ISSUE THREAD

**Issue by Spikeone** [Return-To-The-Roots/s25client#92]
Wenn man etwas "sammeln" möchte in einem Lager, lässt sich das nicht immer abstellen. Erst ein "Einlagern verbieten" und wieder aufheben löst das Problem

Philipp (5-qail-c) wrote on 2014-07-31:	#1
Das gleiche Problem hatte ich auch schon. Leider wurden dann die Waren hin und her geschickt. Teilweise wurden auch Einstellungen zum ein/auslagern vom Programm gesetzt, ohne mein Zutun.

https://bugs.launchpad.net/s25rttr/+bug/1319080

**Comment by BlOedsin** [comment]
I can confirm that there is still an odd behaviour. For easier bugfixing I added my screenshot and a savegame.
I made a screenshot and saved the game, after loading, the settings of the warehouse was different! I think I read in another issue, that it might just be displayed in the wrong way? I think the situation can be created by having a view items set for collecting and then change all for a different mode (In my case I had beer, shields and swords set for collecting, and then used stop all and empty all). 

Savegame: https://www.sendspace.com/file/ram53d 

Before Loading:
![settings](https://cloud.githubusercontent.com/assets/14259828/12376896/dc5dea3c-bd09-11e5-8aeb-2a19a2c9549d.PNG)

After Loading

![settings after loading](https://cloud.githubusercontent.com/assets/14259828/12376895/dc5a0494-bd09-11e5-9663-d4b6a1695b39.PNG)


In the 2nd savegame I assume, that the stroring happens, because the path to the next warehouse is too full and therefore it is stored in that warehouse although storing is forbidden? -->This behavior could be improved, cause there are other roads with more distance available, but not used, but would not be a bug.
Savegame2: https://www.sendspace.com/file/o4jqvm
